partcad/src/partcad/part_factory_sweep.py

```
@@ -0,0 +1,178 @@
+                if ":" in self.source_sketch_name:
+                    self.source_project_name, self.source_sketch_name = resolve_resource_path(
+                        source_project.name,
+                        self.source_sketch_name,
+                    )
+                else:
+                    self.source_project_name = source_project.name
+            self.source_sketch_spec = self.source_project_name + ":" + self.source_sketch_name
+
+            self._create(config)
+
+    async def instantiate(self, part):
```

Ruff (0.8.2)

Missing return type annotation for public function `instantiate`

(ANN201)